### PR TITLE
tasks/cephfs: vstart_runner fixes

### PR DESCRIPTION
**Pull request by jcsp**
Mixture of things that tripped up while running tests all together on a vstart cluster

### CODE REVIEW

File: tasks/cephfs/test_pool_perm.py [hunk 1]
```diff
@@ -80,6 +80,7 @@ def test_forbidden_modification(self):
 
         self.mount_a.umount_wait()
         self.mount_a.mount()
+        self.mount_a.wait_until_mounted()
```
Comment (ajarr) [inline tasks/cephfs/test_pool_perm.py]: So I guess we can be more careful and start using the 'mounted' context manager more widely across the code base and not worry too much about races and if we forgot to umount?
So just,
with self.mount_a.mounted():
as done in 'test_journal_migration.py'

File: tasks/cephfs/test_cap_flush.py [hunk 2]
```diff
@@ -4,9 +4,10 @@
 from textwrap import dedent
 from unittest import SkipTest
 from tasks.cephfs.fuse_mount import FuseMount
-from tasks.cephfs.cephfs_test_case import CephFSTestCase
+from tasks.cephfs.cephfs_test_case import CephFSTestCase, long_running
```
Comment (ajarr) [inline tasks/cephfs/test_cap_flush.py]: do you think I can change the long_running to something like {with, for, use}_teuthology as a follow up patch?

File: tasks/cephfs/test_pool_perm.py (hunk 1 above)
Comment (jcsp) [inline tasks/cephfs/test_pool_perm.py]: Yes, we should make more use of that

**Comment by ajarr** [comment]
Code looks good to me.

**Pull request merged**